tools/importer/import.js

```diff
@@ -185,6 +185,26 @@ function setPlayIcons(main, document) {
   });
 }
 
+// javascript function to fix heading levels in a page, to not skip levels
+function fixHeadings(rootEl) {
+  const headings = [...rootEl.querySelectorAll('h2, h3, h4, h5, h6')];
+  headings.forEach((heading) => {
+    console.log(heading.outerHTML);
```

Comment:
i think we don't need logging

Reply:
fixed in [e21ef66](https://github.com/hlxsites/macquarie-help-centre/pull/94/commits/e21ef6657a364ba3fa8af55c85b99b2af479de8c)